frontend/src/components/common/StudentDetailsInfoPanel.vue

```diff
@@ -413,4 +411,8 @@ export default {
   .v-data-table /deep/ tr td:nth-child(8) {
     width: 12%;
   }
+
```

Comment:
Another option instead of display: none is setting a blank bottom template
`<v-data-table
  ...
>
  <template #bottom></template>
</v-data-table>`